src/Components/Home/Users.js

```diff
@@ -36,7 +38,20 @@ function Users({ list }) {
           filteredList.length > 0 &&
           filteredList.map((user, key) => (
             <Link to={user.username} key={`avatar-${key}`}>
-              <Chip image={user.avatar} className="p-m-2" label={user.name} />
+              <Chip
+                image={user.avatar}
+                className="p-m-2"
+                label={user.name}
+                onImageError={(error) => {
```

Comment:
Since we use this function twice, how about creating a helper function for this.

Reply:
yeah, that would be good, any common path where such function can be moved?

Reply:
I am not able to find any common file where all such utility functions can be placed, how about creating a `src/utils.js` file? From there we can import all such functions.

Reply:
Yep! Feel free to create src/utils.js

Reply:
I have created a helper function for default image behavior.